commands/wip.command.js

```diff
@@ -1,37 +1,58 @@
-const { addLabel, removeLabel } = require('../helpers/label.helper');
-const { listIssueLabels } = require('../helpers/listLabels.helper');
+const { addLabel, removeLabel } = require("../helpers/label.helper");
+const { listIssueLabels } = require("../helpers/listLabels.helper");
 
 const WIPCommand = async context => {
 	const { title } = context.payload.issue || context.payload.pull_request;
 	const labelsFromIssues = await listIssueLabels(context);
 
 	const wipLabel = labelsFromIssues.data.find(
-		label => label.name === ':construction: WIP'
+		label => label.name === ":construction: WIP"
+	);
+	const foundReadyForReviewLabel = labelsFromIssues.data.find(
+		label => label.name === ":mag: Ready for Review"
```

Comment:
Codacy has a fix for the issue: [Replace `↹↹label` with `····(label)`](https://app.codacy.com/gh/aayushchugh/bot/pullRequest?prid=10060852)
```suggestion
    (label) => label.name === ":mag: Ready for Review"
```